tests/testTriggers.go

```diff
@@ -571,6 +571,38 @@ func TriggerDeployNewApps(contexts *[]*scheduler.Context, recordChan *chan *Even
 				UpdateOutcome(event, err)
 			}
 		}
+
+		if len(*contexts) > 2 {
+			mid := len(*contexts) / 2
+			for i, ctx := range *contexts {
+				appVolumes, err := Inst().S.GetVolumes(ctx)
+				UpdateOutcome(event, err)
+				var volumeSpecUpdate *opsapi.VolumeSpecUpdate
+				if i < mid {
+					volumeSpecUpdate = &opsapi.VolumeSpecUpdate{
+						IoThrottleOpt: &opsapi.VolumeSpecUpdate_IoThrottle{
```

Comment:
Do it only on few FIO volumes only . Can you measure how much IOPS and Bandwidth consistently getting on these FIO volumes (may be check for 15 minutes and then throttle IOPS and Bandwidth to it's 50% so that we throttle to the correct number.

Reply:
will take as part of different when how to verify this is cleared